src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -1,22 +1,566 @@
+        return {
+          ...formData,
+          links: linksObject,
+        }
+      })
+    },
+    [formData.links]
+  )
+
+  const updateLink = useCallback(
+    (type, index, updatedUrl, updatedLabel) => {
+      const linksObject =
+        type === COMMUNITY_LINK_TYPE
+          ? {
+              ...formData.links,
+              community: formData.links.community.map((link, i) =>
+                i === index ? { link: updatedUrl, label: updatedLabel } : link
+              ),
+            }
+          : {
+              ...formData.links,
+              documentation: formData.links.documentation.map((link, i) =>
+                i === index ? { link: updatedUrl, label: updatedLabel } : link
+              ),
+            }
+
+      setFormData(formData => {
+        return {
+          ...formData,
+          links: linksObject,
+        }
+      })
+    },
+    [formData.links]
+  )
+
+  const handleOnAssetAdded = useCallback((type, base64, fileExtension) => {
+    setFormData(formData => {
+      return {
+        ...formData,
+        [type]: base64,
+        [`${type}Extension`]: fileExtension,
+      }
+    })
+  }, [])
+
+  const handleOnDaoLogoLoaded = useCallback(
+    (base64, fileExtension) => {
+      handleOnAssetAdded(DAO_LOGO, base64, fileExtension)
+    },
+    [handleOnAssetAdded]
+  )
+
+  const handleOnMobileLogoLoaded = useCallback(
+    (base64, fileExtension) => {
+      handleOnAssetAdded(DAO_LOGO_MOBILE, base64, fileExtension)
+    },
+    [handleOnAssetAdded]
+  )
+
+  const handleOnTokenLogoLoaded = useCallback(
+    (base64, fileExtension) => {
+      handleOnAssetAdded(TOKEN_LOGO, base64, fileExtension)
+    },
+    [handleOnAssetAdded]
+  )
+
+  const handleOnAssetRemoved = useCallback(type => {
+    setFormData(formData => {
+      return {
+        ...formData,
+        [type]: '',
+      }
+    })
+  }, [])
+
+  const handleOnDaoLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(DAO_LOGO)
+  }, [handleOnAssetRemoved])
+
+  const handleOnMobileLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(DAO_LOGO_MOBILE)
+  }, [handleOnAssetRemoved])
+
+  const handleOnTokenLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(TOKEN_LOGO)
+  }, [handleOnAssetRemoved])
+
+  const handleNext = useCallback(() => {
+    onConfigChange('garden', formData)
+    onNext()
+  }, [onConfigChange, onNext, formData])
 
   return (
     <div>
-      {title}
+      <div
+        css={`
+          margin-bottom: ${6 * GU}px;
+          text-align: center;
+        `}
+      >
+        <h1
+          css={`
+            ${textStyle('title1')};
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          Garden Metadata
+        </h1>
+        <div>
+          <p
+            css={`
+              ${textStyle('body1')};
+              color: ${theme.contentSecondary};
+            `}
+          >
+            Fill with your garden information
+          </p>
+        </div>
+      </div>
```

Comment:
Not sure if you have merged with gardens already but we can use the Header form the kit that Paulo added